Makefile

```diff
@@ -15,7 +15,7 @@ verify:
 clean: delete_mgmt_cluster host_cleanup
 
 delete_mgmt_cluster:
-	sudo minikube delete || true
+	sudo su -l -c "minikube delete" "$(USER)"
```

Comment:
I noticed we're not deleting the minikube VM too but I fixed it by just removing the sudo?

Reply:
Hmm actually I see this matches the sudo in `03_launch_mgmt_cluster.sh` - I wonder if we actually need sudo in either place provided the user is in the libvirt group?

Reply:
I'm not sure, I just noticed it wasn't being deleted and made it consistent with everywhere else we're running minikube in metal3-dev-env. I assume if the user is a member of libivrt it would work fine, but maybe this is because of support Ubuntu?

Reply:
The issue is that if you run the whole script in an ubuntu or centos VM that run an off-the-shelf image, the group is created and the user added, but it does not take effect immediately. So not using this sudo su trick would mean that the scripts would fail until you log out and in again.